cosmo_tester/framework/testenv.py

```diff
@@ -265,6 +265,13 @@ def setUpClass(cls):
     def tearDownClass(cls):
         pass
 
+    def wait_for_expected_outputs(self, expected_outputs, timeout=3):
```

Comment:
monitoring data is not the same as output; monitoring data is sent continuously by the diamond agents, whereas a deployment's outputs (the "cloudify meaning of outputs" in this case :)) should be available right after running the install workflow.
i think the wait/timeouts should be removed.